### PR TITLE
Refactor Flux GitHub Action

### PR DESCRIPTION
**Pull request by stefanprodan**
Use composite mode to download and add the `flux` binary to the GitHub runner bin dir.

### CODE REVIEW

File: action/action.yml [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+        tar -C /tmp/flux/ -zxvf /tmp/flux.tar.gz
+    - name: "Add flux binary to /usr/local/bin"
+      shell: bash
+      run: |
+        sudo cp /tmp/flux/flux /usr/local/bin
+    - name: "Cleanup tmp"
+      shell: bash
+      run: |
+        rm -rf /tmp/flux/ /tmp/flux.tar.gz
+    - name: "Verify binary"
```
Comment (hiddeco) [inline action/action.yml:35]:
```suggestion
    - name: "Print flux binary version"
```

maybe? I would expect "verify" to run e.g. `flux check --pre`.

Comment (stefanprodan) [reply action/action.yml:35]: The check command requires kubectl and a Kubernetes cluster. By calling -v we make sure the binary can be run on the runner.

Comment (hiddeco) [reply action/action.yml:35]: I am well aware, that is why my suggestion changed the name of the run, and not the command.

Comment (hiddeco) [reply action/action.yml:35]:
```suggestion
    - name: "Verify correct installation of binary"
```

would be another option. It is more that"verify" is now pretty broad. Anyway, nitpick and non-blocking.

**Review by hiddeco — APPROVED** [review]

File: action/action.yml (hunk 1 above)
Comment (stefanprodan) [reply action/action.yml]: Updated as you suggested ✔️ 

Just noticed that these steps don't show up in GH UI 🙃

**Review by hiddeco — APPROVED** [review]

**Pull request merged**